eng/targets/FunctionalTestAsset.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <Target Name="CollectFunctionalTestPayload" DependsOnTargets="Publish" Returns="@(DependencyPayload)" >
```

Comment:
Unclear how this works any better or more extensively than what's already in Helix.targets for all test projects. Shouldn't need to do anything more than remove `ReferenceOutputAssembly="false"` to get the referenced test asset outputs into the work item payload.